forward43/kickstarter_scraper.py

```diff
@@ -0,0 +1,104 @@
+import os
+
+import json
+import random
+import time
+import requests
+from pathlib import Path
+
+from datetime       import datetime
+from urllib.request import urlopen
+
+
+MAX_VALUE                  = 65536
+KICKSTARTER_URL            = 'https://www.kickstarter.com/discover/advanced.json?'
+KICKSTARTER_DEFAULT_PARAMS = ['sort=popularity']
```

Comment:
Oh, I'm not too sure about this popularity sorting: not sure how their popularity ranking and the random_seed interacts with each other. 
If popularity overpowers the random_seed, we could potentially be getting a lot of projects that overlap between runs, and missing lots of projects as we're only looking at the top 60 popular in that seed.

Reply:
Good point. I just checked that it's not a required parameter. I will get rid of it.
(An alternative would be to use `sort=newest`)